### PR TITLE
[NEXUS-9222] Java 8 Required

### PR DESCRIPTION
**Pull request by bradbeck**
Update the documentation to reflect that Java 8 is now required to run Nexus 3.

https://issues.sonatype.org/browse/NEXUS-9222

### CODE REVIEW

**Comment by joedragons** [comment]
+1

File: chapter-installation.asciidoc [hunk 1]
```diff
@@ -233,10 +233,11 @@ Nexus is fully started once you see a message like the following in the log:
 
 [subs="attributes"]
 ----
-[jetty-main-1] *SYSTEM org.sonatype.nexus.bootstrap.jetty.JettyServer -
+-------------------------------------------------
 
-Started Sonatype Nexus PRO {version-exact}
+Started Sonatype Nexus OSS 3.0.0-b2015091801
```
Comment (mosabua) [inline chapter-installation.asciidoc]: Was this usage of the macro broken?

File: chapter-installation.asciidoc [hunk 2]
```diff
@@ -25,7 +25,7 @@ website].
 You can confirm the installed Java version with the `java` command:
 
 ----
-$java -version
+$ java -version
 java version "1.8.0_45"
```
Comment (mosabua) [inline chapter-installation.asciidoc]: Should we update to a newer version here?

**Comment by mosabua** [comment]
+1 .. but let me merge that into a different branch so we can keep the nexus-3.0.x branch clean

File: chapter-installation.asciidoc (hunk 1 above)
Comment (bradbeck) [inline chapter-installation.asciidoc]: It was more that it said "PRO".

**Comment by mosabua** [comment]
This is merged into a new branch for M6 stuff and will be there until the m6 release.

**Pull request closed without merging**